packages/api/es/search.js

```diff
@@ -23,9 +23,9 @@ class BaseSearch {
     let esConfig;
 
     // this is needed for getting temporary credentials from IAM role
-    if (process.env.MODE === 'local') {
+    if (process.env.TEST) {
```

Comment:
I've written a ticket to address this elsewhere in our codebase, but testing for `process.env.NODE_ENV === 'test'` is going to be more reliable than TEST.  (NODE_ENV='test' will always be true in a test, whether or not we remembered to set TEST.  That addresses the problem of accidentally writing to AWS in our tests.)

https://www.npmjs.com/package/ava#process-isolation